demo/napari_gui/napari_.py

```diff
@@ -0,0 +1,26 @@
+# load data
+from skimage.io import imread
+image = imread('https://samples.fiji.sc/blobs.png')
+
+# start up napari
+import napari
+import pyclesperanto_prototype as cle
+with napari.gui_qt():
+    viewer = napari.Viewer()
+    viewer.add_image(image, name='blobs')
+
+    # push image to GPU
+    input = cle.push_zyx(image)
+
+    # process the mage
+    sigma = 3
+    blurred = cle.gaussian_blur(input, sigma_x=sigma, sigma_y=sigma)
+    binary = cle.threshold_otsu(blurred)
+    labels = cle.connected_components_labeling_box(binary)
```

Comment:
Why box? 😂

Reply:
:P 
Box: 8- resp. 26-connected (3d/2d), partly separable
Diamond: 4- resp. 6-connected, not separable

I engineered this terminology very carefully. And I'd love to know how to improve it.

Reply:
I prefer to use `connected_components_labeling(binary, connectivity={1...ndim})` (see [`ndimage.label`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.label.html)/[`ndimage.generate_binary_structure`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.generate_binary_structure.html#scipy.ndimage.generate_binary_structure)/[`skimage.measure.label`](https://scikit-image.org/docs/dev/api/skimage.measure.html#skimage.measure.label)). Separability is an implementation detail that you can handle internally (when connectivity==ndim).